book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
For the `rentals` instance variable, both the getter and setter methods should be implemented. Kindly use the `attr_accessor` instead of the `atr_reader` to achieve the same.